windows_containers/windows-containers-release-notes-5-x.adoc

```diff
@@ -9,25 +9,32 @@ toc::[]
 [id="about-windows-containers"]
 == About {productwinc}
 
-{productwinc} enables running Windows compute nodes in an {product-title} cluster. Running Windows workloads is possible by using the Red Hat Windows Machine Config Operator (WMCO) to install and manage Windows nodes. With Windows nodes available, you can run Windows container workloads in {product-title}.
+Windows Container Support for Red Hat OpenShift enables running Windows compute nodes in an {product-title} cluster. Running Windows workloads is possible by using the Red Hat Windows Machine Config Operator (WMCO) to install and manage Windows nodes. With Windows nodes available, you can run Windows container workloads in {product-title}.
```

Comment:
The correct product name is `Red Hat OpenShift support for Windows Containers`.

Reply:
Thank you for catching that. Bad cut-and-paste from my original PR that didn't have the change we made to the naming.